From d5f022ecde747fa804f5c3d94a2e60a84fabce63 Mon Sep 17 00:00:00 2001
From: Sreenidhi B R <sreenira@broadcom.com>
Date: Tue, 30 Jul 2013 14:31:52 +0530
Subject: dtre: change default frequency to sub-multiple of 667.

- so that it is in sync with pll and divisor values (as per PRM)
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Nam Ninh <nam.ninh@windriver.com>

diff --git a/arch/mips/boot/dts/xlp2xx-ep-linux.dts b/arch/mips/boot/dts/xlp2xx-ep-linux.dts
index 59b2f1e..0bdb184 100644
--- a/arch/mips/boot/dts/xlp2xx-ep-linux.dts
+++ b/arch/mips/boot/dts/xlp2xx-ep-linux.dts
@@ -108,7 +108,7 @@
 		nae = <250>;
 		sae = <500>;
 		rsa = <500>;
-		dtre = <250>;
+		dtre = <333>;  /* sub-multiple of 667 */
 		cde = <250>;
 	};
 };
diff --git a/arch/mips/boot/dts/xlp2xx-ep-tiny.dts b/arch/mips/boot/dts/xlp2xx-ep-tiny.dts
index 24d13a7..410b8af 100644
--- a/arch/mips/boot/dts/xlp2xx-ep-tiny.dts
+++ b/arch/mips/boot/dts/xlp2xx-ep-tiny.dts
@@ -154,7 +154,7 @@
 		nae = <250>;
 		sae = <500>;
 		rsa = <500>;
-		dtre = <250>;
+		dtre = <333>;  /* sub-multiple of 667 */
 		cde = <250>;
 	};
 };
diff --git a/arch/mips/boot/dts/xlp2xx-helinux.dts b/arch/mips/boot/dts/xlp2xx-helinux.dts
index 649b419..b5f2b2e 100644
--- a/arch/mips/boot/dts/xlp2xx-helinux.dts
+++ b/arch/mips/boot/dts/xlp2xx-helinux.dts
@@ -158,7 +158,7 @@
 		nae = <250>;
 		sae = <500>;
 		rsa = <500>;
-		dtre = <250>;
+		dtre = <333>;  /* sub-multiple of 667 */
 		cde = <250>;
 	};
 };
diff --git a/arch/mips/boot/dts/xlp2xx-linux-dpi.dts b/arch/mips/boot/dts/xlp2xx-linux-dpi.dts
index fd547b5..447acbb 100644
--- a/arch/mips/boot/dts/xlp2xx-linux-dpi.dts
+++ b/arch/mips/boot/dts/xlp2xx-linux-dpi.dts
@@ -103,7 +103,7 @@
 		nae = <250>;
 		sae = <500>;
 		rsa = <500>;
-		dtre = <250>;
+		dtre = <333>;  /* factor of 667 */
 		cde = <250>;
 	};
 };
diff --git a/arch/mips/boot/dts/xlp2xx-linux.dts b/arch/mips/boot/dts/xlp2xx-linux.dts
index bd4355a..f9566f7 100644
--- a/arch/mips/boot/dts/xlp2xx-linux.dts
+++ b/arch/mips/boot/dts/xlp2xx-linux.dts
@@ -105,7 +105,7 @@
 		nae = <250>;
 		sae = <500>;
 		rsa = <500>;
-		dtre = <250>;
+		dtre = <333>;  /* sub-multiple of 667 */
 		cde = <250>;
 	};
 };
diff --git a/arch/mips/boot/dts/xlp2xx-multi-os.dts b/arch/mips/boot/dts/xlp2xx-multi-os.dts
index 99608c4..24c906f 100644
--- a/arch/mips/boot/dts/xlp2xx-multi-os.dts
+++ b/arch/mips/boot/dts/xlp2xx-multi-os.dts
@@ -214,7 +214,7 @@
 		nae = <250>;
 		sae = <500>;
 		rsa = <500>;
-		dtre = <250>;
+		dtre = <333>;  /* sub-multiple of 667 */
 		cde = <250>;
 	};
 };
diff --git a/arch/mips/boot/dts/xlp2xx-netos.dts b/arch/mips/boot/dts/xlp2xx-netos.dts
index ae26e13..29e6411 100644
--- a/arch/mips/boot/dts/xlp2xx-netos.dts
+++ b/arch/mips/boot/dts/xlp2xx-netos.dts
@@ -96,7 +96,7 @@
 		nae = <250>;
 		sae = <500>;
 		rsa = <500>;
-		dtre = <250>;
+		dtre = <333>;  /* sub-multiple of 667 */
 		cde = <250>;
 	};
 };
diff --git a/arch/mips/boot/dts/xlp2xx-tiny.dts b/arch/mips/boot/dts/xlp2xx-tiny.dts
index c0b87ec..6eca7ee 100644
--- a/arch/mips/boot/dts/xlp2xx-tiny.dts
+++ b/arch/mips/boot/dts/xlp2xx-tiny.dts
@@ -181,7 +181,7 @@
 		nae = <250>;
 		sae = <500>;
 		rsa = <500>;
-		dtre = <250>;
+		dtre = <333>;  /* sub-multiple of 667 */
 		cde = <250>;
 	};
 };
-- 
1.7.1

